### ISSUE TITLE
Quotes are considered HTML in fields

### ISSUE THREAD

**Issue by Jnesselr** [Hoektronics/BotQueue#130]
Our protection of HTML injection also stops quotes (both single and double quote) and probably a few other things because they're escaped, and so don't match the original input. That's at least what I think is happening.

**Comment by eric116** [comment]
I just noticed this a few days ago. It wouldn't let me enter / and - in an error description.

**Comment by Jnesselr** [comment]
Correct. I'm working on a newer version that doesn't validate that on
submit but escapes it on output.

**Comment by eric116** [comment]
Sweet.
